homeassistant/components/todoist/calendar.py

```diff
@@ -317,11 +330,7 @@ def name(self) -> str:
     async def async_update(self) -> None:
         """Update all Todoist Calendars."""
         await super().async_update()
-        await self.data.async_update()
-        # Set Todoist-specific data that can't easily be grabbed
-        self._cal_data["all_tasks"] = [
-            task[SUMMARY] for task in self.data.all_project_tasks
-        ]
+        self._apply_coordinator_update()
```

Comment:
This logic was just moved into `_apply_coordinator_update` so it could be shared here and in the call in `_handle_coordinator_update`.